hack/jenkins/e2e-runner.sh

```diff
@@ -174,16 +174,7 @@ function install_google_cloud_sdk_tarball() {
 
 # Figures out the builtin k8s version of a GCI image.
 function get_gci_k8s_version() {
-    local -r image_description=$(gcloud compute images describe ${KUBE_GCE_MASTER_IMAGE} --project=${KUBE_GCE_MASTER_PROJECT})
-    # Staged GCI images all include versions in their image descriptions so we
-    # extract builtin Kubernetes version from them.
-    local -r k8s_version_re='.*Kubernetes: ([0-9a-z.-]+),.*'
-    if [[ ${image_description} =~ ${k8s_version_re} ]]; then
-        local -r gci_k8s_version="v${BASH_REMATCH[1]}"
-    else
-        echo "Failed to determine builtin k8s version for image ${image_name}: ${image_description}"
-        exit 1
-    fi
+    local -r gci_k8s_version="v$(gsutil cat gs://container-vm-image-staging/k8s-version-map/${KUBE_GCE_MASTER_IMAGE})"
```

Comment:
In this context, if `JENKINS_GCI_IMAGE_FAMILY` is set, `KUBE_GCE_MASTER_IMAGE` will always be overridden to the latest GCI build, disregarding what user sets.

https://github.com/kubernetes/kubernetes/blob/master/hack/jenkins/e2e-runner.sh#L236
https://github.com/kubernetes/kubernetes/blob/master/hack/jenkins/e2e-runner.sh#L197